setup.py

```diff
@@ -95,5 +97,6 @@ def build_extension(self, ext):
     cmdclass=dict(build_ext=CMakeBuild),
     extras_require={
         "benchmarking": benchmarking,
+        "all": segyio,
```

Comment:
If you put it under "all", I'd add benchmarking to the list, too. In any case, I'd always move this to setup.cfg.